src/AppInstallerCLICore/Commands/FeaturesCommand.cpp

```diff
@@ -53,7 +53,7 @@ namespace AppInstaller::CLI
             {
                 table.OutputLine({
                     std::string{ feature.Name() },
-                    Resource::Loader::Instance().ResolveString(ExperimentalFeature::IsEnabled(feature.GetFeature()) ? Resource::String::FeaturesEnabled : Resource::String::FeaturesDisabled),
```

Comment:
In this PR, `Loader` class is made private to prevent directly resolving strings which could potentially result in unresolved placeholders